python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -301,6 +290,69 @@ cpdef write_parquet(
     else:
         return None
 
+
+cdef class ParquetChunkedWriter:
+    """
+    ParquetChunkedWriter lets you write out a series of Tables to a single Parquet file
+
+    See Also
+    --------
+    cudf.io.parquet.write_parquet
+    """
+    cdef shared_ptr[pq_chunked_state] state
+    cdef cudf_io_types.sink_info sink
+    cdef cudf_io_types.statistics_freq stat_freq
+    cdef cudf_io_types.compression_type comp_type
+    cdef object index
+
+    def __cinit__(self, path, index=None, compression=None, statistics="ROWGROUP"):
```

Comment:
```suggestion
    def __cinit__(self, string path, object index=None, object compression=None, string statistics="ROWGROUP"):
```

Reply:
I don't want to change path to a ```str``` - this code will also work with Path objects, so I just declared as an object instead. Aside from that I've changed to use the type changes like you suggested

Reply:
Perfect, I wasn't aware of that stipulation. Thanks!